documentation/0019-package-json/index.md

```diff
@@ -0,0 +1,492 @@
+    "postcss-loader": "^2.0.8",
+    "postcss-url": "^7.2.1",
+    "rimraf": "^2.6.0",
+    "semver": "^5.3.0",
+    "shelljs": "^0.7.6",
+    "uglifyjs-webpack-plugin": "^1.1.1",
+    "url-loader": "^0.5.8",
+    "vue-jest": "^1.0.2",
+    "vue-loader": "^13.3.0",
+    "vue-style-loader": "^3.0.1",
+    "vue-template-compiler": "^2.5.2",
+    "webpack": "^3.6.0",
+    "webpack-bundle-analyzer": "^2.9.0",
+    "webpack-dev-server": "^2.9.1",
+    "webpack-merge": "^4.1.0"
+  },
+  "engines": {
+    "node": ">= 6.0.0",
+    "npm": ">= 3.0.0"
+  },
+  "browserslist": ["> 1%", "last 2 versions", "not ie <= 8"]
+}
+```
+
+there are _lots_ of things going on here:
+
+- `name` sets the application/package name
+- `version` indicates the current version
+- `description` is a brief description of the app/package
+- `main` set the entry point for the application
+- `private` if set to `true` prevents the app/package to be accidentally published on `npm`
+- `scripts` defines a set of node scripts you can run
+- `dependencies` sets a list of `npm` packages installed as dependencies
+- `devDependencies` sets a list of `npm` packages installed as development dependencies
+- `engines` sets which versions of Node this package/app works on
+- `browserslist` is used to tell which browsers (and their versions) you want to support
+
+All those properties are used by either `npm` or other tools that we can use.
+
+## Properties breakdown
+
+This section describes the properties you can use in detail. We refer to "package" but the same thing applies to local applications which you do not use as packages.
+
+Most of those properties are only used on the <https://www.npmjs.com/>, other by scripts that interact with your code, like `npm` or others.
+
+### `name`
+
+Sets the package name.
+
+Example:
+
+```json
+"name": "test-project"
+```
+
+The name must be less than 214 characters, must not have spaces, it can only contain lowercase letters, hyphens (`-`) or underscores (`_`).
+
+This is because when a package is published on `npm`, it gets its own URL based on this property.
+
+If you published this package publicly on GitHub, a good value for this property is the GitHub repository name.
+
+### `author`
+
+Lists the package author name
+
+Example:
+
+```json
+{
+  "author": "Flavio Copes <flavio@flaviocopes.com> (https://flaviocopes.com)"
+}
+```
+
+Can also be used with this format:
+
+```json
+{
+  "author": {
+    "name": "Flavio Copes",
+    "email": "flavio@flaviocopes.com",
+    "url": "https://flaviocopes.com"
+  }
+}
+```
+
+### `contributors`
+
+As well as the author, the project can have one or more contributors. This property is an array that lists them.
+
+Example:
+
+```json
+{
+  "contributors": [
+    "Flavio Copes <flavio@flaviocopes.com> (https://flaviocopes.com)"
+  ]
+}
+```
+
+Can also be used with this format:
+
+```json
+{
+  "contributors": [
+    {
+      "name": "Flavio Copes",
+      "email": "flavio@flaviocopes.com",
```

Comment:
I think this is your actual personal email address which is available on your website. 
Just heads up, lot of people go through the guide and some of them may not be so nice. Putting the actual email ID may not be a good call IMHO

Reply:
Yep good idea! Changed it 👍